.github/workflows/node.js.yml

```diff
@@ -5,7 +5,7 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [ main, prod ]
   pull_request:
```

Comment:
Is this saying on all pull requests it is running dev deployment? We may not want that as it can screw up our CF and cause problems. In the future if we go to the moon we can add tests and run that at that step.

Reply:
it will build and run tests on any pull request to those branches, but it won't run the deploy since there is an if statement checking the branch name before the deployment process